requirements.txt

```diff
@@ -1,3 +1,5 @@
+# 1. Don't include forget to populate spcific requirements.txt. IMORTANT!!!
```

Comment:
Ha-ha. You are right) This sentence will likely disappear from here soon,
let's ignore it meanwhile)

On Tue, Jun 23, 2015, 07:53 Lakshmi Kannan <notifications@github.com> wrote:

> In requirements.txt
> <https://github.com/StackStorm/st2/pull/1647#discussion_r33009336>:
>
> > @@ -1,3 +1,5 @@
> > +# 1. Don't include forget to populate spcific requirements.txt. IMORTANT!!!
>
> Nit: typo. specific. IMPORTANT.
> Nit: Sentence isn't making sense.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StackStorm/st2/pull/1647/files#r33009336>.
>